specification/monitor/resource-manager/readme.md

```diff
@@ -42,6 +42,37 @@ directive:
     reason: There are open issues (bugs) in the validator affecting some of the examples and since there is no way to selectively disable the validation for a particular example or paths, all of the example validation is being turned off.
 ```
 
+### Tag: package-2022-04
+
+These settings apply only when `--tag=package-2022-04` is specified on the command line
+
+``` yaml $(tag) == 'package-2022-04'
```

Comment:
Could we also change the default tag to this new tag? Please make sure the difference between this and previous tag (2022-02) is your new swagger - 2022-04-01/actionGroups

Reply:
@ArcturusZhang , @jianyexi , could you folks please advise what is the next step here? This has been pending for a while. The LintDiff errors were not related to my changes. Thank you!